package.json

```diff
@@ -91,7 +91,7 @@
     "babel-runtime": "^6.26.0",
     "backbone": "1.2.1",
     "clipboard": "1.6.1",
-    "handlebars": "4.0.5",
+    "handlebars": "4.0.11",
```

Comment:
Now that we are checking in a `yarn.lock` file, are we OK with making this a flexible version?